tests/util/fixture_fluent.py

```diff
@@ -109,15 +109,17 @@ def launch_fluent_solver_2ddp():
         container_dict = {"host_mount_path": pyfluent.USER_DATA_PATH}
         file_transfer_service = RemoteFileTransferStrategy()
         solver_session = pyfluent.launch_fluent(
-            version="2d",
-            precision="double",
+            version=pyfluent.SolverVersion.TWO_DIMENSIONAL,
```

Comment:
The Fluent launcher calls it _Dimension_. _Version_ is a heavily overloaded term so this can get confusing and error-prone for users.

How about:
```
solver_2d = pyfluent.launch_fluent(dimension=pyfluent.DIMENSION.TWO)
solver_3d = pyfluent.launch_fluent(dimension=pyfluent.DIMENSION.THREE)
```
Meanwhile, we could support `version` for backward compatibility.

Reply:
Absolutely agree. Will update it soon.

Reply:
N.b. `ONE` and `TWO` without any superfluous suffixes.